Source/OxyPlot/Series/ContinuousHistogramItem.cs

```diff
@@ -0,0 +1,120 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="ContinuousHistogramItem.cs" company="OxyPlot">
+//   Copyright (c) 2014 OxyPlot contributors
+// </copyright>
+// <summary>
+//   Represents an item in a RectangleSeries.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OxyPlot.Series
+{
+    /// <summary>
+    /// Represents an item in a <see cref="ContinuousHistogramSeries" />, a bin (range) and its area.
+    /// </summary>
+    public class ContinuousHistogramItem
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ContinuousHistogramItem" /> class.
+        /// </summary>
+        /// <param name="rangeStart">The range start.</param>
+        /// <param name="rangeEnd">The range end.</param>
+        /// <param name="area">The area.</param>
+        /// <param name="color">The color.</param>
+        public ContinuousHistogramItem(double rangeStart, double rangeEnd, double area, OxyColor color)
+        {
+            this.RangeStart = rangeStart;
+            this.RangeEnd = rangeEnd;
+            this.Area = area;
+            this.Color = color;
+        }
+        
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ContinuousHistogramItem" /> class.
+        /// </summary>
+        /// <param name="rangeStart">The range start.</param>
+        /// <param name="rangeEnd">The range end.</param>
+        /// <param name="area">The area.</param>
+        public ContinuousHistogramItem(double rangeStart, double rangeEnd, double area)
+        {
+            this.RangeStart = rangeStart;
+            this.RangeEnd = rangeEnd;
+            this.Area = area;
+
+            this.Color = OxyColors.Automatic;
+        }
+
+        /// <summary>
+        /// Gets or sets the color of the item.
+        /// </summary>
+        /// <remarks>If the color is not specified (default), the color of the series will be used.</remarks>
+        public OxyColor Color { get; set; }
```

Comment:
do we really need this?

Reply:
Probably not, I'll strip it out (forgot it was there, to be quite honest)